lib/core/ui/root_widget.dart

```diff
@@ -34,57 +36,53 @@ Widget injectUMEWidget({
   enable
       ? PluggableMessageService().resetListener()
       : PluggableMessageService().clearListener();
-  return _FloatingWidget(
-      child: child,
-      enable: enable,
-      supportedLocales: supportedLocales,
-      localizationsDelegates: localizationsDelegates);
+  WidgetsFlutterBinding.ensureInitialized();
+  WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
+    if (enable) {
+      final overlayEntry = OverlayEntry(builder: (BuildContext context) {
+        return _FloatingWidget();
+      });
+      overlayKey.currentState?.insert(overlayEntry);
+    }
+    StaticVariableManager.hasAttached = enable;
+  });
+  if (!enable) return child;
+  return Directionality(
+    textDirection: TextDirection.ltr,
+    child: Stack(
+      children: <Widget>[
+        RepaintBoundary(child: child, key: rootKey),
+        MediaQuery(
+          data: MediaQueryData.fromWindow(WidgetsBinding.instance!.window),
```

Comment:
Should we use the `MediaQueryData` from `ui.window` here?

Reply:
https://api.flutter.dev/flutter/dart-ui/window.html
Maybe `WidgetsBinding.instance!.window` is more suitable?